docs/basic/simulate.md

```diff
@@ -1,15 +1,17 @@
-模拟。顾名思义，就是用计算机来模拟题目中要求的操作，比如 NOIP 2014 的 [生活大爆炸版石头剪刀布](https://uoj.ac/problem/15) ，只需要按照题面的意思来写就可以了。
+Simulation, as the name implies, is the operation that computers use to simulate the process required in the question. Similar to NOIP 2014's [Big Bang version rock paper scissors] (https://uoj.ac/problem/15) (original link in Chinese), coding according to the meaning of the title would be fine.
 
-当然，模拟并不总是很好写，参见经典题目 [魔兽世界](http://bailian.openjudge.cn/practice/3750/) 和 [猪国杀](https://loj.ac/problem/2885) 。
+Of course, simulations are not always easy to write. See classic questions like [World of Warcraft] (http://bailian.openjudge.cn/practice/3750/) and [pig kingdom kill] (https://loj.ac/problem /2885) for reference.
 
-模拟题目通常具有码量大、操作多、思路繁复的特点。并且由于它码量大，会导致很难查错，如果在考试中写错是相当浪费时间的。
+Simulation questions usually have large code size, many operations, and complicated ideas. And because of its large amount of code, the debugging process is really difficult. If you write errors during the competition, it would be quite a waste of time.
```

Comment:
思路 logic？

write error -> made mistakes
```suggestion
Simulation questions usually have large code size, many operations, and complicated logics. And because of its large amount of code, the debugging process is really difficult. Therefore, it would be quite time-consuming if you made mistakes during contests.
```